apps/chat-e2e/src/tests/sidePanelEntityDragAndDrop.test.ts

```diff
@@ -306,6 +308,7 @@ dialTest(
       async () => {
         prompt = promptData.prepareDefaultPrompt();
         await dataInjector.createPrompts([prompt]);
+        await localStorageManager.setCollapsedSection('Organization');
```

Comment:
need to collapse prompt Organization sections here-> set promptCollapsedSections=["Organization"]